packages/grafana-ui/src/components/DateTimePickers/DateTimePicker/DateTimePicker.tsx

```diff
@@ -97,7 +104,7 @@ export const DateTimePicker = ({
     strategy: 'fixed',
   });
 
-  const handleCalendarApply = useCallback(
+  const onApply = useCallback(
```

Comment:
Just want to share the reasoning behind `handleX` vs `onX` (Absolute not a suggestion of change).

The reasoning is very simple, a `handleX` is a handler that is passed as a prob, and `onX` is a prop.
It makes it very easy to know what is what and you don't have to make up namesNames when there is both a prop and a handler in the same component :)

Reply:
yep i understand the reasoning and i mostly agree, i just don't think this is the place to implement that. it muddies the diff with renaming 😅 believe there's a lint rule we can enable if we want to implement this across the codebase 👍